.gitignore

```diff
@@ -1,6 +1,9 @@
 # Directories/files that may be generated by this project
 build
 coverage
+cypress/videos
```

Comment:
I suppose these are useful to have when debugging a failing test? Do they come with some overhead to the test run time? Can they be disabled, and if so, should that be the default? Or at least considered for a CI context.

Reply:
Yes, this could be turned off with a config option (or an env variable), but I do think it makes sense to keep it even in a CI context. The e2e tests can fail for mysterious reasons in CI and work in local and having videos somewhere could be great to debug those issues, don't you think?

Anyway, I don't mind disabling those by default and see if we need them later